client/packages/common/src/ui/discovery/DiscoveredServers.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {
+  AlertIcon,
+  Box,
+  HomeIcon,
+  InlineSpinner,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  MenuList,
+  Typography,
+  useTranslation,
+  frontEndHostDisplay,
+  CheckboxEmptyIcon,
+  FrontEndHost,
+  useNativeClient,
+  GqlProvider,
+  QueryClientProvider,
+  QueryClient,
+  frontEndHostGraphql,
+  useInitialisationStatus,
+  InitialisationStatusType,
+} from '@openmsupply-client/common';
+
+type ConnectToServer = ReturnType<typeof useNativeClient>['connectToServer'];
+
+type DiscoverServersProps = {
+  servers: FrontEndHost[];
+  connect: ConnectToServer;
+  discoveryTimedOut: boolean;
+};
+
+export const DiscoveredServers = ({
+  servers,
+  connect,
+  discoveryTimedOut,
+}: DiscoverServersProps) => {
+  const t = useTranslation('app');
```

Comment:
useElectronClient used to be here, relocated upstream, hmm, could have kept it here, can revert